### ISSUE TITLE
WGLMakie recorded animations keep the initial value of some observables

### ISSUE THREAD

**Issue by ederag** [MakieOrg/Makie.jl#3268]
With `WGLMakie v0.8.14` on linux, in a standalone Pluto notebook + update,
the lines are updated, but not the title (all the frames display `t = 0.0`).
The notebook has been adapted from the [docs](https://docs.makie.org/stable/explanations/animation/index.html#animations_using_observables).
It works fine with `CairoMakie` (without the JSServe cell).

I know that there is an upcoming version that might fix this (lot of work on WGLMakie),
so this is more like an mwe for testing purposes.

[Makie_pluto_animation.zip](https://github.com/MakieOrg/Makie.jl/files/12776814/Makie_pluto_animation.zip)

**Comment by ederag** [comment]
The issue showed up while creating an animation, but it happens any time the observable
is used inside one of the plots kwargs (passed as a positional argument it works just fine).

**Comment by SimonDanisch** [comment]
MWE seems to be:
```julia

  t = Observable("text1")
  fig = text(Point2f(0), text =t)
  display(fig)
  t[] = "text2"

```
Some other plot functions with kwargs + observable to seem to work like expected though

**Comment by ederag** [comment]
The recorded animation is fixed as well with #3275, thanks !
